nixos/modules/services/misc/spice-webdavd.nix

```diff
@@ -0,0 +1,34 @@
+{ config, pkgs, lib, ... }:
+
+with lib;
+let
+  cfg = config.services.spice-webdavd;
+in
+{
+  options = {
+    services.spice-webdavd = {
+      enable = mkEnableOption "Spice guest webdav proxy daemon";
+    };
+  };
+
+  config = mkIf cfg.enable {
+    # pretty useless without these
+    services.davfs2.enable = true;
+    services.avahi.enable = true;
```

Comment:
doesn't avahi have other things enabled by default that you may not want? there's not much of a better way to do this but it shouldn't interact with other things. (And it probably won't, since `nssmdns` is `false`-by-default. Maybe add a warning/assertion?)

Reply:
I checked more carefully and our `phodav` doesn't have Avahi (which, AFAICT, is what needs DBus) support compiled in anyway. Therefore I just took all the Avahi and DBus stuff out.